cli/daemon/daemon.go

```diff
@@ -87,8 +90,29 @@ func runDaemonCommand(cmd *cobra.Command, args []string) {
 	logrus.Infof("Starting daemon on TCP port %s", port)
 	lis, err := net.Listen("tcp", fmt.Sprintf(":%s", port))
 	if err != nil {
-		logrus.Fatalf("failed to listen: %v", err)
+		// Invalid port, such as "Foo"
+		var dnsError *net.DNSError
+		if errors.As(err, &dnsError) {
+			logrus.Errorf("Failed to listen on TCP port: %s. %s is unknown name.", port, dnsError.Name)
```

Comment:
can we do `feedback.Errorf` instead? This way the error will be always visible in the shell, not regarding of passing `-v` and will be added to the logs as well. Same for other occurrences of `logrus.*`

Reply:
Thank you for the feedback. I have made the changes based on your request and left the [original `logrus.Fatalf("failed to serve: %v", err)`](https://github.com/arduino/arduino-cli/pull/592/files#diff-aab5babf75b671a8b8ccb9fcc6b09f46R118) as is.